sdk/eventhubs/azure-messaging-eventhubs/src/main/resources/eventhubs-messages.properties

```diff
@@ -4,11 +4,11 @@ REQUEST_VALUE_NOT_VALID=Back pressure request value not valid. It must be betwee
 EVENT_DATA_DOES_NOT_FIT=EventData does not fit into maximum number of batches. '%s'
 CANNOT_SEND_EVENT_BATCH_EMPTY=Cannot send an EventBatch that is empty.
 ERROR_SENDING_BATCH=Error sending batch.
-FAILED_TO_CLAIM_OWNERSHIP=Failed to claim ownership of partition {} - {}
-LOAD_BALANCING_FAILED=Load balancing for event processor failed - {}
+FAILED_TO_CLAIM_OWNERSHIP=Failed to claim ownership
+LOAD_BALANCING_FAILED=Load balancing for event processor failed.
```

Comment:
Nit since we added a period here that we could add them to the rest of the sentences.